BnbnavNetClient/Services/TextToSpeech/DummyTextToSpeechProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BnbnavNetClient.Services.TextToSpeech;
+class DummyTextToSpeechProvider : ITextToSpeechProvider
+{
+    public Task SpeakAsync(string text, CultureInfo culture)
+    {
+        return Task.CompletedTask;
+    }
+}
```

Comment:
```suggestion
    public CultureInfo Culture { get; set; }
}
```

Make the culture a global thing